tests/common/test_common_constants.py

```diff
@@ -0,0 +1,143 @@
+"""
+Sample data for testing Common functions in pyracf.
+"""
+
+from typing import Union
+
+import tests.test_utilities as TestUtilities
+
+
+def get_sample(sample_file: str, function_group: str) -> Union[str, bytes]:
+    return TestUtilities.get_sample(sample_file, function_group)
```

Comment:
Should we move test sample files specific to these 'common' tests to 'tests/common' so that it is clear what that these test samples are scoped/specific to the 'common' tests?

Reply:
sure. I avoided that in the past as there can be some samples used for both "common" testing and others, but it is probably a good idea. Done.

Reply:
I can make it so that no data is duplicated, but it may make the common constants file a bit messy indicating it pulls test data from different places.

Reply:
There is clearly a tradeoff here, but I do think that a little bit of test data duplication here is okay since it makes our test code easier to understand since it is clearer where test data is coming from.

However, a possible solution to this problem to this problem is to maybe use symbolic links maybe? I think git can handle them, but I don't know if they would work properly on Windows, so at the end of the day it might just be better to just have a little bit of data duplication. Let me know what you think.

Reply:
Sounds good, we can definitely put a streamlining of test data/cases on the pipeline for the future.